src/components/Navbar/Navbar.css

```diff
@@ -1,59 +1,54 @@
-.dcl.navbar {
+.dui-navbar2 {
```

Comment:
This change should be in all classNames and inside the component too
```suggestion
.dui-navbar {
```